vars/piperPipelineStageInit.groovy

```diff
@@ -23,6 +23,10 @@ import static com.sap.piper.Prerequisites.checkScript
      * Defines the library resource containing the container map.
      */
     'containerMapResource',
+    /**
+     * Provides a clone from the specified repository.
+     */
+    'checkoutMap',
```

Comment:
Is General the right place. Isn't that only meant to be configured as parameter?

Reply:
Yes, configuring it as parameter is sufficient 👍.